### DISCUSSION TITLE
Would there be any future updates for the nogizaka message app?

### DISCUSSION THREAD

**Discussion by 64600013** [proshunsuke/colmsg#65] [General]
Since the new message app for nogizaka is now released, is there any plans on supporting it as well?

**Comment by proshunsuke** [comment]
I've been so busy recently that it's been hard to find the time, but I'm going to make an effort to support it.

**Comment by proshunsuke** [comment]
Released [`v3.0.0`](https://github.com/proshunsuke/colmsg/releases/tag/v3.0.0)
